src/jit/emitxarch.cpp

```diff
@@ -48,6 +48,15 @@ bool IsSSEOrAVXInstruction(instruction ins)
 #endif // !FEATURE_AVX_SUPPORT
 }
 
+bool IsAVXExcludingSSEInstruction(instruction ins)
```

Comment:
Standard function header missing.

May be this should be renamed to IsAVXOnlyInstruction()?